auction/prices.js

```diff
@@ -0,0 +1,92 @@
+const CoinGecko = require('coingecko-api');
```

Comment:
Is this duplicated 1:1 from oracle/prices.js? We should probably pull prices.js out of both and refactor kava-tools to use the shared features. For now, let's make an issue for `timebox: refactor package or rewrite in go`.